layouts/shortcodes/sponsors.html

```diff
@@ -0,0 +1,40 @@
+<div class="sponsor-images">
+    <a href="https://www.chanzuckerberg.com">
+        <img src="/images/logos/CZI.svg" alt="Logo of the Chan Zuckerberg Initiative" />
+    </a>
+    <a href="https://www.tidelift.com">
+        <img src="/images/logos/tidelift.png" width="200" alt="Logo of Tidelift" />
+    </a>
+    <a href="https://numfocus.org">
+        <img src="/images/logos/NumFOCUS_sponsored_project_logo.svg" width="200" alt="Logo of NumFOCUS" />
```

Comment:
NumFOCUS itself does not give us enough direct funding to list it here. I'd rather do it the same way as done on https://numpy.org/about/

Reply:
Ok I thought the SDG grants and the additional donations would be enough to have them here.

Reply:
Donations aren't coming from NumFOCUS, they just play the role of fiscal host. So with SDG's we're getting max $15k/yr if we get a grant every round. At least according to the numpy guidelines, that's below the bar to be listed.